src/status_im/ui/screens/accounts/styles.cljs

```diff
@@ -50,7 +50,7 @@
 
 (def account-badge-pub-key-text
   {:font-size      14
-   :letter-spacing -0.2
+   :ios            {:letter-spacing -0.2}
```

Comment:
I think you should use `defstyle` not `def` to do platform specific styling

Reply:
:+1:

Reply:
good point, didn't know it existed :) 
fixed